tests/integration/test_github.py

```diff
@@ -169,6 +169,19 @@ async def test_get_authenticated_user(self, codecov_vcr):
             "scope": "read:org,repo:status,user:email,write:repo_hook",
         }
 
+    @pytest.mark.asyncio
+    async def test_get_authenticated_user_with_public_emails(self, codecov_vcr):
+        code = "71367b9f258ca9a60c44"
+
+        handler = Github(
+            oauth_consumer_token=dict(
+                key="Iv23liSqj8DAO20A3KLA",
+                secret="a6a6397fffea369e54495c88ca469d988ea4ccd2",
```

Comment:
Quick thing though, are these the legitimate keys/secrets? If so, now that you have the cassette, you can get rid of these/fake them so that no one knows about the secrets themselves. If they are, and you wanna go the extra mile, try rebasing the commit where these were introduced so there's no track of these committed in gh